packages/core/src/browser/addEventListener.spec.ts

```diff
@@ -33,4 +33,27 @@ describe('addEventListener', () => {
       expect(zoneJsPatchedRemoveEventListener).not.toHaveBeenCalled()
     })
   })
+
+  describe('Untrusted event', () => {
+    it('should be ignored if __ddIsTrusted is false', () => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about adding a test for ignored when `__ddIsTrusted` is absent?